example/simple/src/main.c

```diff
@@ -245,6 +356,13 @@ int main(int argc, char *argv[])
 	if (initialize_cache(ctx, &cache1))
 		error("Unable to start cache\n");
 
+	/*
+	 * Increase cache refcount. Otherway cache will be destroyed
+	 * on cache stop preventing destroying of management queue,
+	 * and thus leading to memory leak.
+	 */
+	ocf_mngt_cache_get(cache1);
```

Comment:
Let's integrate https://github.com/Open-CAS/ocf/pull/144 first and remove this get/put

Reply:
Right! :)